### DISCUSSION TITLE
how to use the visitor count feature

### DISCUSSION THREAD

**Discussion by dataml-cn** [uPagge/uBlogger#97] [Q&A]
I want to use the visitors count feature, but I don’t know how to configure it

i find the code in `/layouts/_default/summary.html`, but no `params.views` in simpleSite `config.toml`: 

```html
{{- with .Params.Views -}}                    
<span class="meta-item post-meta-views">
<span class="svg-icon icon-eye"&gt;&lt;/span>{{ . }} 
</span>
{{- end -}}
```

**Comment by uPagge** [comment]
Anything that is not a bug will be envied in discussions

**Comment by uPagge** [comment]
The setup guide will appear in the new documentation, I hope I will have time this month :)
